examples/asset/embedded_asset.rs

```diff
@@ -1,5 +1,11 @@
 //! Example of loading an embedded asset.
 
+//! An embedded asset is an asset included in the program's memory, in contrast to other assets that are normally loaded from disk to memory when needed.
+//! The below example embeds the asset at program startup, unlike the common use case of embedding an asset at build time. Embedded an asset at program startup can be useful
```

Comment:
TBH, we should probably demonstrate the typical usage instead.

Reply:
Yeah I agree, but I don't know how to do it :(